openworld/sdk/generator/client/__main__.py

```diff
@@ -81,12 +81,19 @@ def generate_code(
     for visitor_path in visitors_path:
         module = dynamic_load_module(visitor_path)
         if hasattr(module, "visit"):
-            visitors.append(module.visit)
+            # If visitor was given a positive order, it will be parsed,
+            # otherwise it shall have an order later than ordered visitors.
+            visitor_order = int(module.order) if hasattr(module, "order") and module.order > 0 else int(1e18)
+
+            visitors.append((visitor_order, module.visit))
         else:
             raise Exception(f"{visitor_path.stem} does not have any visit function")
 
+    # Sort visitors by their given order
```

Comment:
Comments on lines 92&95 do not add any value, we may remove them.

Reply:
I will.